### DISCUSSION TITLE
Error with XCODE15 - IOS17 and cordova plugins

### DISCUSSION THREAD

**Discussion by vesvello** [driftyco/ionic-native#4663] [Q&A]
Im getting this error:

Assertion failed: (aliasSectionNum == sectionNum && "alias and its target must be located in the same section"), function assignAliasAtomOffsetInSection, file Layout.cpp, line 3248

I tried this solution but it doesnt work for me:

Build Settings -> Other Linker Flags -> -ld64

Any help with this?

**Comment by lcrumling** [comment]
I'm having the same issue. I also tried adding -ld64, I also tried -ld_prime and -ld_classic; nothing has worked so far.
